unit_tests/tests/sensor/test_flex_sensor.cpp

```diff
@@ -0,0 +1,148 @@
+#include "flex_sensor.h"
+#include "engine_test_helper.h" // Used for timesourcing
+#include "global.h"
+
+#include <gtest/gtest.h>
+
+class FlexSensorForTests : public ::testing::Test
+{
+
+protected:
+
+    FlexSensorForTests () :
+            dut ()
```

Comment:
explicit no-arg constructor is not required

Reply:
I guess no constructor is required at all, the default is plenty here.

Reply:
Noted